pyproject.toml

```diff
@@ -10,10 +10,13 @@ flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
 gunicorn = "^20.0.4"
+pymongo = {extras = ["srv"], version = "^3.12.0"}
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.2"
 selenium = "^3.141.0"
+pymongo = {extras = ["srv"], version = "^3.12.0"}
```

Comment:
This line should be deleted - it's already in the dependencies list above (as it should be)

Reply:
Removed from dev-dependencies